psalm-baseline.xml

```diff
@@ -718,11 +674,14 @@
     <ArgumentTypeCoercion>
       <code><![CDATA[$options['pluginManager']]]></code>
     </ArgumentTypeCoercion>
+    <InvalidArgument>
+      <code>array_values($processedParts)</code>
+    </InvalidArgument>
```

Comment:
Anything worth investigating here?

Reply:
Nah. This is psalm better understanding how little it understands this existing code. My current plan is to get native types everywhere and whittle down the baseline as much as poss for v3, so it's something I should get to later on.